README-pt.md

```diff
@@ -15,21 +15,21 @@ Após concluir esta jornada, o leitor saberá como:
 Deseja levar seu aplicativo do Watson ao próximo nível? Quer aproveitar os ativos da marca Watson? Participe do programa [With Watson](https://www.ibm.com/watson/with-watson), que oferece recursos técnicos, de marketing e da marca exclusivos para amplificar e acelerar sua solução comercial incorporada do Watson. 
 
 ## Componentes inclusos 
-- Bluemix Watson Conversation 
-- Bluemix Watson Discovery
-- Banco de dados NoSQL Bluemix Cloudant 
+- IBM Cloud Watson Conversation 
+- IBM Cloud Watson Discovery
+- Banco de dados NoSQL IBM Cloud Cloudant 
```

Comment:
"IBM Cloud Cloudant" doesn't sound right.  Could be just Cloundant or IBM Cloundant. The longer name in docs is "IBM® Cloudant® NoSQL DB for IBM Cloud", but usually it is shortened.